### PR TITLE
cyberpanel installion

### PR DESCRIPTION
**Pull request by hakandemiral**
I have read and understood the Contributor's Certificate of Origin available at the end of 
https://raw.githubusercontent.com/hetzneronline/community-content/master/tutorial-template.md
and I hereby certify that I meet the contribution criteria described in it.
Signed-off-by: Hakan Demiral hakan.98@outlook.com

### CODE REVIEW

File: tutorials/how-to-install-cyberpanel/01.en.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
++ 1GB or above Ram.
++ 10GB Disk Space.
+
+## Step 2 - Installion
+### Step 2.1 - System update
+When you set up a new server, system components such as programs and kernels are usually not available at the latest. Let's update all components and system on the server, enter the following commands sequentially.
+
+**CentOS**
+```bash
+sudo yum upgrade
```
[the remark this answers is not in the document]
Comment (hakandemiral) [reply tutorials/how-to-install-cyberpanel/01.en.md:44]: yes i found out when i searched, thanks

**Comment by hakandemiral** [comment]
@rook59 these are ok

**Review by rook59 — APPROVED** [review]

**Comment by leojakeson** [comment]
This is unfortunately much too short to be accepted as a tutorial.

I've formatted the tutorial to fit with our requirements.

Please expand on the current steps, and consider adding a few more steps on how to configure CyberPanel once it has been installed. Also consider adding a few images.

Once that has been done we can look into merging this.

**Comment by rook59** [comment]
Do you still plan to work on this tutorial @hakandemiral?

**Comment by rook59** [comment]
If you want to finish this tutorial please let us know and we can reopen your PR. :)

**Pull request closed without merging**